core/templates/dev/head/css/oppia.css

```diff
@@ -245,10 +245,14 @@ body {
   width: calc(100% - 280px);
 }
 
+/* Added overflow:hidden; to this class to fix a problem with the screen flickering between 2 and 3 columns
```

Comment:
Great comment! Please:

(a) add a link to this issue thread
(b) (nit) put quotation marks around "overflow: hidden;" -- viz., `Added "overflow: hidden;" to this class ...`
(c) ensure lines are at most 80 characters long.